keyboards/pegasus/keymaps/vial/config.h

```diff
@@ -0,0 +1,5 @@
+/* SPDX-License-Identifier: GPL-2.0-or-later */
```

Comment:
`vial` keymap has to be completely removed

Reply:
Thanks for the quick review! I removed `vial` keymap, included `BOOTLOADER` to rules.mk, and committed your suggestions. Please let me know if there might be anything else I should revise. Thanks again!